src/auto_pickup.cpp

```diff
@@ -371,7 +371,7 @@ void auto_pickup::test_pattern(const int iTab, const int iRow)
     std::vector<std::string> vMatchingItems;
     std::string sItemName = "";
 
-    if (vRules[iTab][iRow].sRule == "") {
+    if ( !vRules[iTab][iRow].sRule.empty() ) {
```

Comment:
This changes the meaning, there is a new `!` at the front.

Reply:
Thanks! I missed this when reviewed by myself.